Update tidb-lightning-overview.md (#7514) (#7676)
pingcap/docs: tidb-lightning/tidb-lightning-overview.md

Currently, TiDB Lightning supports:

> - Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-aurora-to-tidb.md).

Reviewer: ```suggestion
- Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-from-aurora-using-lightning.md).
```

## Restrictions

- If you use TiDB Lighting together with TiFlash:

>     No matter a table has TiFlash replica(s) or not, you can import data to that table using TiDB Lightning. Note that this might slow the TiDB Lightning procedure, which depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.
> - For Apache Parquet files, TiDB Lightning currently only accepts Amazon Aurora Parquet files.

Reviewer: ```suggestion
    No matter a table has TiFlash replica(s) or not, you can import data to that table using TiDB Lightning. Note that this might slow the TiDB Lightning procedure, which depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.

- For Apache Parquet files, TiDB Lightning currently only accepts Amazon Aurora Parquet files.
```

## Restrictions

- If you use TiDB Lighting together with TiFlash:

    No matter a table has TiFlash replica(s) or not, you can import data to that table using TiDB Lightning. Note that this might slow the TiDB Lightning procedure, which depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.

>     
> 

Reviewer: ```suggestion

```
